<launch>

   <param name="img_w" value="640" type="int"/>
   <param name="img_h" value="480" type="int"/>
   <param name="hight_min" value="0.5" type="double"/>
   <param name="hight_max" value="1.0" type="double"/>
   <param name="kx" value="0.005" type="double"/>
   <param name="ky" value="0.005" type="double"/>
   <param name="kz" value="0.005" type="double"/>
   <param name="v_max" value="0.5" type="double"/>
   <param name="kyaw_rate" value="0.005" type="double"/>


   <!-- 以下带port的参数名是给BT节点用的 -->
   <param name="stop_hover_port" value="false" type="bool"/>
   <param name="stop_detect_port" value="false" type="bool"/>
   <param name="hover_is_end_port" value="false" type="bool"/>

   <node pkg="mission_pkg" type="bt_ros" name="single_hit" output="screen">
    <param name="config_path" value="$(find mission_pkg)/config/single_hitballon.xml" type="str"/>
  </node>

</launch>
